app/src/main/res/values/strings.xml

```diff
@@ -100,6 +104,7 @@
 
   <!--Send Geometry -->
   <string name="send_geometry_toolbar_title">Send Geometry</string>
+  <string name="send_geometry_invalid_input_message" translatable="false">@string/send_quadrilateral_invalid_input_message</string>
```

Comment:
I already said I think it's common to both Send...Fragments. Anyway, if not- this should point to @string/invalid_input_message no?